uint8[] req
int32 round
---
uint8[] resp
int32 cid
int32 round
